src/injected/focus-change-handler.ts

```diff
@@ -47,13 +47,17 @@ export class FocusChangeHandler {
 
     private automatedChecksDataIsPopulated(storeData: TargetPageStoreData): boolean {
         return (
+            storeData.cardSelectionStoreData &&
+            storeData.unifiedScanResultStoreData &&
             storeData.cardSelectionStoreData.focusedResultUid !== null &&
```

Comment:
nit: Can these null checks all use the same syntax? (either `if(thing)` or `if(thing !== null)` for all three) (same comment for lines 59-61 in this file)